dev/packages/beats/system-0.0.1/manifest.yml

```diff
@@ -9,6 +9,7 @@ categories:
 - logs
 - metrics
 release: ga
+removable: true
```

Comment:
Nit: This is where it is false. But I think we can do this as a manual adjustment.

Reply:
Fixed.

If this is only for the system package, I guess that this option could be inverted - single property for non-removable/locked/etc.